b2b-sample/guardio-insurance-sample-application/components/settingsComponents/identity-providers/buttonGroupIdentityProviderDetails.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Trash from '@rsuite/icons/Trash';
+import { useSession } from 'next-auth/react';
+import React from 'react';
+import { Button, IconButton, Stack, useToaster } from 'rsuite';
+import decodeDeleteIdentityProvider 
+from '../../../util/apiDecode/settings/identityProvider/decodeDeleteIdentityProvider.JS';
+import {successTypeDialog, errorTypeDialog} from '../../util/dialog'
+
+export default function ButtonGroupIdentityProviderDetails(props) {
+
+    const { data: session } = useSession();
+    const toaster = useToaster();
+
+    const onIdpDelete = (response) => {
+        if (response) {
+            successTypeDialog(toaster, "Success", "Identity Provider Deleted Successfully");
```

Comment:
`Identity provider is deleted successfully`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95